core/settings/qPrefGeocoding.h

```diff
@@ -17,9 +17,9 @@ class qPrefGeocoding : public QObject {
 	static qPrefGeocoding *instance();
 
 	// Load/Sync local settings (disk) and struct preference
-	void loadSync(bool doSync);
-	void load() { loadSync(false); }
-	void sync() { loadSync(true); }
+	static void loadSync(bool doSync);
+	static void load() { loadSync(false); }
+	static void sync() { loadSync(true); }
```

Comment:
This does make the commit **title** a bit misleading - that one had me expect only whitespace changes.
Not critical, but just something I noticed.

Reply:
hmmm, this should have been in one of the earlier commits, whitespace should be only whitespace, but sometimes git rebase --continue outtrixes me, let me have a look